tests/tas-elasticsearch/src/test/java/org/alfresco/elasticsearch/reindexing/NodesSecondaryAncestorIndexingTests.java

```diff
@@ -103,6 +104,7 @@ public void testSecondaryAncestorWithDocumentAsSecondaryChild()
     }
 
     @Test(groups = TestGroup.SEARCH)
```

Comment:
After rethinking shouldn't it be `@Test(groups = TestGroup.SEARCH, disabled = true)` 🤔